charts/helm-project-operator/templates/deployment.yaml

```diff
@@ -6,7 +6,9 @@ metadata:
   labels: {{ include "helm-project-operator.labels" . | nindent 4 }}
     app: {{ template "helm-project-operator.name" . }}
 spec:
-  replicas: 1
+  {{- if .Values.replicas }}
+   replicas: {{ .Values.replicas }}
```

Comment:
```suggestion
  replicas: {{ .Values.replicas }}
```

Reply:
extra space

Reply:
sadness -- it's always the spaces that get me